api/src/main/resources/liquibase-update-update-to-latest.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
```

Comment:
@WolfSchlegel i looked at the comments but i still do not yet understand the need to have "update" twice in the  file name. :)